telemetry/definitions/commonDefinitions.json

```diff
@@ -5777,6 +5787,21 @@
                 }
             ]
         },
+        {
+            "name": "experiment_feature_config",
```

Comment:
is experiment_ the right prefix or should it be toolkit_, aws_, or something like that

Reply:
I will update this to `aws_`